src/core/ext/filters/client_channel/xds/xds_client.cc

```diff
@@ -896,15 +897,22 @@ void XdsClient::ChannelState::AdsCallState::AcceptLdsUpdate(
   }
   if (GRPC_TRACE_FLAG_ENABLED(grpc_xds_client_trace)) {
     gpr_log(GPR_INFO,
-            "[xds_client %p] LDS update received: route_config_name=%s, "
-            "cluster_name=%s",
+            "[xds_client %p] LDS update received: route_config_name=%s",
             xds_client(),
             (!lds_update->route_config_name.empty()
                  ? lds_update->route_config_name.c_str()
-                 : "<inlined>"),
-            (lds_update->rds_update.has_value()
-                 ? lds_update->rds_update->cluster_name.c_str()
-                 : "<to be obtained via RDS>"));
+                 : "<inlined>"));
+    if (lds_update->rds_update.has_value()) {
+      gpr_log(GPR_INFO, "  RouteConfiguration contains %lu routes",
+              lds_update->rds_update.value().routes.size());
```

Comment:
The line 907 and 970 has a format error shows up during the import and break many tests:
format specifies type 'unsigned long' but the argument has type 'std::__1::vector<grpc_core::XdsApi::RdsRoute, std::__1::allocator<grpc_core::XdsApi::RdsRoute> >::size_type' (aka 'unsigned int')